### ISSUE TITLE
Nitpick: Fog Light Propagation on a wall torch is as if on a ground torch. (Blockstates are disregarded)

### ISSUE THREAD

**Issue by slushieuwu** [X0nk/Bliss-Shader#354]
### Describe the bug
Plain and simple, Fog Light Propagation doesn't care about blockstates.
![image](https://github.com/user-attachments/assets/068f0049-dd7d-448b-bb79-3cd13c615b07)

### Describe how your minecraft is set up
![image](https://github.com/user-attachments/assets/468190de-b199-4221-a2b3-db165398a0c2)

### Describe how to reproduce the bug
Simply enable Floodfill and Fog Light Propagation, then place a wall torch, or any light block with different blockstates.

Sorry to nitpick over something so minute :p

**Comment by X0nk** [comment]
yeah the floodfill had no need to be so detailed with models for its colored light, so when the light propagation came around none of that was considered.
